packages/kbn-doc-links/src/get_doc_links.ts

```diff
@@ -879,6 +879,7 @@ export const getDocLinks = ({ kibanaBranch, buildFlavor }: GetDocLinkOptions): D
       roleAndPrivileges: `${FLEET_DOCS}fleet-roles-and-privileges.html`,
       proxiesSettings: `${FLEET_DOCS}fleet-agent-proxy-support.html`,
       unprivilegedMode: `${FLEET_DOCS}elastic-agent-unprivileged.html#unprivileged-change-mode`,
+      httpMonitoring: `${FLEET_DOCS}elastic-agent-monitoring-configuration.html`,
```

Comment:
@kilfoyle We need to consolidate or duplicate the information on these two pages somehow:

https://www.elastic.co/guide/en/fleet/current/elastic-agent-monitoring-configuration.html

https://www.elastic.co/guide/en/fleet/current/agent-policy.html#change-policy-enable-agent-monitoring

With this PR, all the options from the first page are now available to be configured in the UI.

Based on the original issue description, https://github.com/elastic/kibana/issues/153950#issue-1645981330, it seems the first page is not capturing all the available options under `agent.monitoring` either.

When this is done we also need to remove this section in favor of whatever documentation is added about the full advance monitoring options:
https://www.elastic.co/guide/en/fleet/current/agent-policy.html#agent-policy-http-monitoring

Reply:
Thanks @jen-huang! I've opened [this issue](url) for the docs.

I'll come back to you soon with a URL for that UI link.

Reply:
I'd suggest that we just point the UI link to the second URL:
https://www.elastic.co/guide/en/fleet/current/agent-policy.html#change-policy-enable-agent-monitoring

I'll update that section in the docs to include the various configuration options, and I'll also add the missing ones into the first link (`https://www.elastic.co/guide/en/fleet/current/elastic-agent-monitoring-configuration.html`). That first link is specifically for standalone agent, and we duplicate the setting descriptions so that they appear in both that section and in the Fleet-managed section of the docs, i.e., the second link.

Does that sound okay?

Reply:
Sounds good! I updated the link used here.